tests/foreman/api/test_hostcollection.py

```diff
@@ -15,446 +15,457 @@
+
+    :expectedresults: The host collection was updated with a new host.
+
+    :CaseImportance: Critical
+    """
+    hosts = fake_hosts
+    host_collection = entities.HostCollection(host=[hosts[0]], organization=module_org).create()
+    host_collection.host_ids = [hosts[1].id]
+    host_collection = host_collection.update(['host_ids'])
+    assert host_collection.host[0].id == hosts[1].id
+
+
+@pytest.mark.upgrade
+@pytest.mark.tier1
+def test_positive_update_hosts(module_org, fake_hosts):
+    """Update host collection's hosts.
+
+    :id: 0433b37d-ae16-456f-a51d-c7b800334861
+
+    :expectedresults: The host collection was updated with new hosts.
+
+    :CaseImportance: Critical
+    """
+    hosts = fake_hosts
+    host_collection = entities.HostCollection(host=hosts, organization=module_org).create()
+    new_hosts = [entities.Host(organization=module_org).create() for _ in range(2)]
+    host_ids = f"""[{','.join([f"{host.id}" for host in new_hosts])}]"""
+    host_ids = host_ids.strip('][').split(',')
+    host_collection.host_ids = host_ids
+    host_collection = host_collection.update(['host_ids'])
+    assert {host.id for host in host_collection.host} == {host.id for host in new_hosts}
+
+
+@pytest.mark.upgrade
+@pytest.mark.tier1
+def test_positive_delete(module_org):
+    """Check if host collection can be deleted
+
+    :id: 13a16cd2-16ce-4966-8c03-5d821edf963b
+
+    :expectedresults: Host collection was successfully deleted
+
+    :CaseImportance: Critical
+    """
+    host_collection = entities.HostCollection(organization=module_org).create()
+    host_collection.delete()
+    with pytest.raises(HTTPError):
+        host_collection.read()
+
+
+@pytest.mark.parametrize('name', **parametrized(invalid_values_list()))
+@pytest.mark.tier1
+def test_negative_create_with_invalid_name(module_org, name):
+    """Try to create host collections with different invalid names
+
+    :id: 38f67d04-a19d-4eab-a577-21b8d62c7389
+
+    :parametrized: yes
+
+    :expectedresults: The host collection was not created
+
+    :CaseImportance: Critical
+    """
+    with pytest.raises(HTTPError):
+        entities.HostCollection(name=name, organization=module_org).create()
+
+
+@pytest.mark.tier1
+def test_positive_add_remove_subscription(
+    module_org, module_lce, fake_hosts, module_promoted_cv, module_activation_key
+):
+    """Try to bulk add and remove a subscription to members of a host collection.
+
+    :id: c4ec5727-eb25-452e-a91f-87cafb16666b
+
+    :steps:
+
+        1. Create HC, add AK to HC
+        2. Create product so we can use it's subscription
+        3. Create some VMs and register them with AK so they are in HC
+        4. Add the subscription to the members of the Host Collection
+        5. Assert subscription is added
+        6. Bulk remove subscription
+        7. Assert it is removed
+
+    :expectedresults: subscription added to, and removed from, members of host collection
+
```

Comment:
I think Docstring `:parametrized: yes` is also needed here as well.

Reply:
but there is no `@pytest.mark.parametrize` decorator above line 402